x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/enrollment/tool/BootstrapPasswordAndEnrollmentTokenForInitialNodeTests.java

```diff
@@ -123,13 +119,14 @@ public void setup() throws Exception {
             .setSecureSettings(secureSettings)
             .put("path.home", tempDir)
             .build();
+        CommandUtils commandUtils = mock(CommandUtils.class);
     }
 
     public void testGenerateNewPasswordSuccess() throws Exception {
         terminal.addSecretInput("password");
         String includeNodeEnrollmentToken = randomBoolean() ? "--include-node-enrollment-token" : "";
         String output = execute(includeNodeEnrollmentToken);
-        assertThat(output, containsString("elastic user password: Aljngvodjb94j8HSY803"));
+        assertThat(output, containsString("elastic user password: "));
```

Comment:
we can wrap CommandLineUtils in a method in BootstrapPasswordAndEnrollmentTokenForInitialNode so that we can override it for tests, but I don't think it's worth it